library/Ivoz/Provider/Domain/Service/Extension/ExtensionFactory.php

```diff
@@ -42,6 +44,19 @@ public function fromMassProvisioningCsv(
                 ExtensionInterface::ROUTETYPE_USER
             );
 
+        if ($countryPrefix) {
+            $extensionDto
+                ->setNumberCountryId(
+                    substr($countryPrefix, 1)
```

Comment:
NumberCountry is a foreign key to Country table. You'll have to map country prefix to it's pk